### DISCUSSION TITLE
Welcome to curriculum Discussions!

### DISCUSSION THREAD

**Discussion by safaldas** [Planet-NULLCAST/curriculum#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by that-coder** [comment]
Hey folks

I am Jayakrishnan a.k.a thatcoder, Software Engineer at Neoito and Member of Nullcast Ninjas. Happy to contribute in this amazing initiative - a free curriculum for web developers (built by industry leaders) and Thanks to everyone supporting this.

- JK
